src/cartogram_info/cartogram_info.cpp

```diff
@@ -78,7 +78,7 @@ void CartogramInfo::replace_missing_and_zero_target_areas()
   // Calculate threshold for small target areas. For GeoDivs below this
   // threshold, the target area is scaled up for easier calculation.
   const double small_target_area_threshold =
-    total_target_area_with_data * small_area_threshold_frac;
+    total_target_area_with_data * 0.001;
```

Comment:
Instead of changing here, I suggest you change the variable `small_area_threshold_frac` in the `constants.h` file. Also, it will be great if you could provide the reasoning behind choosing 0.001 instead of any other arbitrary value.